src/drivers/batt_smbus/batt_smbus.cpp

```diff
@@ -250,6 +59,7 @@ int serial_number();
 BATT_SMBUS::BATT_SMBUS(int bus, uint16_t batt_smbus_addr) :
 	I2C("batt_smbus", "/dev/batt_smbus0", bus, batt_smbus_addr, 100000),
 	_enabled(false),
+	_last_report{},
```

Comment:
I would recommend adding default initializers in the class definition instead. It's then a lot easier to see that everything is initialized, and you don't have to fight with ordering.